### PR TITLE
MAC-128 Updated content to DfE in agreements related pages

### PR DESCRIPTION
**Pull request by ChrisJWoodcock**

### CODE REVIEW

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAgreement/WhenDoYouWantToView.cshtml [hunk 1]
```diff
@@ -105,106 +105,3 @@
       });
     </script>
 }
-
-
-
-@*<div class="govuk-grid-row">
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAgreement/WhenDoYouWantToView.cshtml:111]: what is all this commented code?

File: src/SFA.DAS.EmployerAccounts.Web/Views/Transfers/TransferConnectionInvitationAuthorization.cshtml [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 @if (Model.AuthorizationResult.IsAuthorized)
 {
-    <p>Before starting a connection, both the sending and receiving employers need to read and understand the <a href="https://www.gov.uk/government/publications/apprenticeship-funding-and-performance-management-rules-2017-to-2018" class="govuk-link" rel="nofollow">rules for sending and receiving transfers</a>.</p>
+    <p>Before starting a connection both the sending and receiving employers need to read and understand the <a href="https://www.gov.uk/government/publications/apprenticeship-funding-and-performance-management-rules-2017-to-2018" class="govuk-link" rel="nofollow">rules for sending and receiving transfers</a>.</p>
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts.Web/Views/Transfers/TransferConnectionInvitationAuthorization.cshtml:11]: I know its in the AC but does is this correctly punctuated without the comma?

**Review by cofaulco — COMMENTED** [review]
couple of questions

File: src/SFA.DAS.EmployerAccounts.Web/Views/Transfers/TransferConnectionInvitationAuthorization.cshtml (hunk 2 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.EmployerAccounts.Web/Views/Transfers/TransferConnectionInvitationAuthorization.cshtml:11]: I've just asked an English teacher and she says that both are technically correct but that she prefers the green one

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAgreement/WhenDoYouWantToView.cshtml (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAgreement/WhenDoYouWantToView.cshtml:111]: From the GitHub history it looks like it was commented out whilst the CDN work was being done in 2019 and not removed - so I've junked it now...

**Review by cofaulco — APPROVED** [review]

**Pull request merged**